cmd/argocd/commands/repo.go

```diff
@@ -43,6 +44,8 @@ func NewRepoAddCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
 		sshPrivateKeyPath              string
 		insecureIgnoreHostKey          bool
 		insecureSkipServerVerification bool
+		tlsClientCertPath              string
+		tlsClientCertKeyPath           string
 		enableLfs                      bool
 	)
 	var command = &cobra.Command{
```

Comment:
might be nice to include an example using `Example: `

Reply:
Yes, and I should also update the docs. Will write something up. With `Example:`, you mean in the Cobra command definition?

Reply:
correct,  I don't think you _need_ to create docs

Reply:
Well, working software before extensive documentation BUT I think docs for user facing functionality is somewhat important.

Reply:
minor, but I think you should just do this using back-quotation

Reply:
Fixed.